sdk-setup/src/mb2

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 #
-# Copyright (C) 2013-2019 Jolla Ltd.
-# Copyright (C) 2019-2020 Open Mobile Platform LLC.
+# Copyright (C) 2013 - 2020 Jolla Ltd.
+# Copyright (C) 2019 - 2020 Open Mobile Platform LLC.
```

Comment:
hm, isn't it more common to omit spaces here?

Reply:
The examples in sage's presentation have spaces